### PR TITLE
Library update

### PR DESCRIPTION
**Pull request by tigerw**
* Updated luaexpat and expat
* Updated zlib
* Updated JsonCPP

### CODE REVIEW

**Comment by SphinxC0re** [comment]
It works? Then go and squash them and rebase them onto the current master

**Comment by tigerw** [comment]
It works. They're already rebased, but since I would say they're logically separate, would it be okay if they weren't squashed into one?

**Comment by SphinxC0re** [comment]
You're right. Since they are indeed logically separate you can leave them as they are

File: .gitmodules [hunk 1]
```diff
@@ -22,12 +22,21 @@
...
+	ignore = dirty
+[submodule "lib/jsoncpp"]
+	path = lib/jsoncpp
+	url = https://github.com/open-source-parsers/jsoncpp.git
+	ignore = dirty
 [submodule "lib/TCLAP"]
 	path = lib/TCLAP
 	url = https://github.com/mc-server/TCLAP.git
+	ignore = dirty
+
```
Comment (SphinxC0re) [inline .gitmodules]: missing newline

**Pull request merged**